iiblib/iib_client.py

```diff
@@ -91,8 +91,8 @@ def add_bundles(
         organization=None,
         overwrite_from_index=False,
         overwrite_from_index_token=None,
-        deprecation_list=None,
         raw=False,
+        deprecation_list=None,
```

Comment:
The commit Change an order of deprecation_list var in add_bundle should be squashed with previous one - no need to have this in single commit

Reply:
The whole PR will be squashed

Reply:
I usually do rebase and merge. But if the plan is to squash everything, then it does not matter.